src/pages/p/[handle]/safe/index.page.tsx

```diff
@@ -0,0 +1,42 @@
+import { AppWrapper } from 'components/common'
+import Loading from 'components/Loading'
+import { ProjectSafeDashboard } from 'components/ProjectSafeDashboard'
+import { V1ProjectContext } from 'contexts/v1/projectContext'
+import { InferGetStaticPropsType } from 'next'
+import { V1UserProvider } from 'providers/v1/UserProvider'
+import { V1ProjectMetadataProvider } from 'providers/v1/V1ProjectMetadataProvider'
+import { V1ProjectProvider } from 'providers/v1/V1ProjectProvider'
+import { useContext } from 'react'
+import { getStaticProps } from '../index.page'
+
```

Comment:
@johnnyd-eth don't we need `getServerSideProps` here?

Like v2

<img width="535" alt="Screen Shot 2022-10-25 at 2 49 44 PM" src="https://user-images.githubusercontent.com/12551741/197681760-aa1fb988-db03-4263-9b35-519eb4d67745.png">

Otherwise how is `  handle,  metadata,` defined in the page function on line 23?

Reply:
@johnnyd-eth static props isn't appropriate here.

I'll take a look.

<img width="751" alt="Screenshot 2022-10-27 at 10 15 37 AM" src="https://user-images.githubusercontent.com/12551741/198159277-154d5fb3-6874-44cf-9c50-4bc712ef810c.png">